### PR TITLE
deb-pkg: rename acrn-sos to acrn-service-vm

### PR DESCRIPTION
**Pull request by hufenglin**
Rename acrn-sos to acrn-service-vm

Tracked-On: #6688

Signed-off-by: Hu Fenglin <fenglin.hu@intel.com>

### CODE REVIEW

**Comment by acrnsi** [comment]
Can one of the admins verify this patch?

**Comment by wenlingz** [comment]
OK to verify doc

File: misc/packaging/100_ACRN [hunk 1]
```diff
@@ -27,8 +27,8 @@ while IFS= read -r line; do
     fi
 done < <(blkid |grep ext4 |grep ${type})
 
-if ls /boot/vmlinuz*acrn-sos* 1> /dev/null 2>&1;then
-    sos_kernel=$(ls -tr /boot/vmlinuz-*acrn-sos* | tail -1)
```
Comment (gvancuts) [inline misc/packaging/100_ACRN:31]: We should update `sos_kernel` to `service_vm_kernel` to improve readibility and be consistent.

**Review by gvancuts — CHANGES_REQUESTED** [review]

File: misc/packaging/100_ACRN (hunk 1 above)
Comment (hufenglin) [reply misc/packaging/100_ACRN:31]: Thanks,  It has been fixed.

**Comment by wenlingz** [comment]
@NanlinXie please help approve this PR, thanks

**Review by gvancuts — APPROVED** [review]

**Review by NanlinXie — APPROVED** [review]

**Comment by szhen11** [comment]
start to run premerge test

**Pull request merged**